docs/help/gettingStartedWithAI.md

```diff
@@ -0,0 +1,93 @@
+
+### Prompts
+
+Prompts provide the models directions on what to generate. As a general rule of thumb, the more detailed your prompt is, the better your result will be.
+
+### Models
+
+Models are the magic that power InvokeAI. These files represent the output of training a machine on understanding massive amounts of images - providing them with the capability to generate new images using just a text description of what you’d like to see. (Like Stable Diffusion!)
+
+Invoke offers a simple way to download several different models upon installation, but many more can be discovered online, including at ****. Each model can produce a unique style of output, based on the images it was trained on - Try out different models to see which best fits your creative vision!
+
+- *Models that contain “inpainting” in the name are designed for use with the inpainting feature of the Unified Canvas*
+
+### Noise
+
+### Scheduler
+
+Schedulers guide the process of removing noise (de-noising) from data. They determine:
+
+1. The number of steps to take to remove the noise.
+2. Whether the steps are random (stochastic) or predictable (deterministic).
+3. The specific method (algorithm) used for de-noising.
+
+Experimenting with different schedulers is recommended as each will produce different outputs!
+
+### Steps
+
+The number of de-noising steps each generation through. 
+
+Schedulers can be intricate and there's often a balance to strike between how quickly they can de-noise data and how well they can do it. It's typically advised to experiment with different schedulers to see which one gives the best results. There has been a lot written on the internet about different schedulers, as well as exploring what the right level of "steps" are for each. You can save generation time by reducing the number of steps used, but you'll want to make sure that you are satisfied with the quality of images produced!
+
+### Low-Rank Adaptations / LoRAs
+
+Low-Rank Adaptations (LoRAs) ****are like a smaller, more focused version of model, intended to focus on training a better understanding of how a specific character, style, or concept looks.
+
+### Embeddings
+
+Embeddings, like LoRAs, assist with more easily prompting for certain characters, styles, or concepts. However, embeddings are trained to more update the relationship between a specific word (known as the “trigger”) and the intended output. Embeddings may sometimes also be referred to as Textual Inversions (TIs).
+
+### ControlNet
+
+ControlNet is a neural network model that can be used to control output from models. This can take many forms, such as controlling poses of people in generated images or providing edges to based image generation on. The impact of the ControlNet can also be adjusted to increase or decrease the similarity of the generated image to the ControlNet.
```

Comment:
I would change to "ControlNets are neural network models that are able to extract key features from an existing image and use these features to guide the output of the image generation model."